src/PKSim.Core/Model/OriginData.cs

```diff
@@ -1,58 +1,134 @@
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using NPOI.HSSF.Record;
 using OSPSuite.Core.Domain;
 using PKSim.Assets;
 
 namespace PKSim.Core.Model
 {
+   public class OriginDataParameter
```

Comment:
I have added this new class. Maybe better as InputParameter. It's always the value and unit. We will need a project conversion but it's much easier. Dynamic parameters coming from DiseaseState are also such parameters